generators/server/templates/build.gradle.ejs

```diff
@@ -319,33 +283,29 @@ dependencies {
     implementation group: "tech.jhipster", name: "jhipster-framework"<% if (reactive) { %>, {
         exclude group: "org.springframework", module: "spring-webmvc"
     }<% } %>
+    implementation "javax.annotation:javax.annotation-api"
 <%_ // This is useful for major Spring Boot migration
```

Comment:
reordered the same way as maven.
Next pr will be to align both in alphabetical order: noticed many different dependencies between both which was not a good sign